datasets/bls/cpsaat18/pipeline.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+  - type: bigquery_table
+    table_id: cpsaat18
+    description: "Current population survey 18: Employed persons by detailed industry, sex, race, and Hispanic or Latino ethnicity"
+
+dag:
+  initialize:
+    dag_id: cpsaat18
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: '2021-06-23'
+    max_active_runs: 1
+    schedule_interval: "@once"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "GoogleCloudStorageToBigQueryOperator"
+      description: "Task to load the CPSAAT18 data to the BigQuery table"
+      args:
+        task_id: "load_csv_to_bq"
+        bucket: "{{ var.json.bls.source_bucket }}"
+        source_objects: ["cpsaat18/2020.csv"]
```

Comment:
csv is 2020 so if we switch to yearly this doesn't work

Reply:
Done! 3c83177